table-attributes.md

```diff
@@ -0,0 +1,124 @@
+
+## Usage
+
+The table attribute is in `key=value` form. If there are multiple attributes, you need to use commas to separate them. The examples are as follows, where `t` is the name of the table to be modified, `p` is the name of the partition to be modified. For the items in `[]`, it is optional to fill them in.
+
++ Set attributes of a table or partition:
+
+    ```sql
+    alter table t [partition p ]attributes[=]'key=value[, key1=value1...]';
+    ```
+
++ Reset attributes of a table or partition:
+
+    ```sql
+    alter table t [partition p ]attributes[=]default;
+    ```
+
++ See all attributes of tables and partitions:
+
+    ```sql
+    select * from information_schema.attributes;
+    ```
+
++ See the attributes configured to a table or partition:
+
+    ```sql
+    select * from information_schema.attributes where id='schema/t[/p]';
+    ```
+
++ See all tables and partitions that have specific attributes:
+
+    ```sql
+    select * from information_schema.attributes where attributes like '%key%';
+    ```
+
+## Attribute override rules
+
+The attribute configured for a table takes effect on all partitions of the table. However, there is one exception: If the table and partition are configured with the same attribute but different attribute values, the partition attribute overrides the table attribute. For example, suppose that the table `t` is configured with the `key=value` attribute, and the partition `p` is configured with `key=value1`.
+
+```sql
+alter table t attributes[=]'key=value';
+alter table t partition p attributes[=]'key=value1';
+```
+
+In this case, the attribute taking actual effect on the partition `p` is `key=value1`.
+
+## Region merging control through the table attribute
+
+### Usage scenario
+
+If there is write hotspot or read hotspot, you can use the table attribute to take control of merging regions by adding the `merge_option` attribute to a table or partition and setting its value to `deny`. The followings are two usage scenarios.
+
+#### Write hotspot on a newly created table or partition
+
+If a hotspot problem occurs when writing data to a newly created table or partition, it is necessary to avoid the problem by splitting and scattering regions. However, if there is a certain time interval between the split and scatter operation and writes, it is impossible to truly avoid the write hotspot. This is because the split operation regarding the new table or partition produces empty regions, so if the time interval occurs, the regions might be merged. In this case, you can solve this merging issue by adding the `merge_option` attribute to the table or partition and setting its value to `deny`.
```

Comment:
Global comment: region --> Region. Please pay attention to capitalization when it refers to a TiDB-specific concept.
```suggestion
If a hotspot issue occurs when data is written to a newly created table or partition, you usually need to split and scatter regions. However, if there is a certain time interval between the split/scatter operation and writes, you do not truly manage to avoid the write hotspot. This is because the split operation performed when the table or partition is created produces empty regions, so if the time interval exists, the split regions might be merged. To handle this case, you can add the `merge_option` attribute to the table or partition and set the attribute value to `deny`.
```

Reply:
“you do not truly manage to avoid the write hotspot.” 这部分修改为 “you are not able to/cannot truly manage to avoid the write hotspot." ，强调解决热点是无法解决的，这样如何呢？

Reply:
好的，已修改 comment